scipy/special/functions.json

```diff
@@ -1402,6 +1402,21 @@
             "cdft2_wrap": "dd->d"
         }
     },
+    "stirling2_inexact": {
+        "stirling2.h++": {
+            "stirling2_inexact": "ii->d"
+        }
+    },
+    "stirling2_temme": {
+        "stirling2.h++": {
+            "stirling2_temme": "ii->d"
+        }
+    },
+    "stirling2_dp": {
+        "stirling2.h++": {
+            "stirling2_dp": "ii->d"
+        }
+    },
```

Comment:
I think we need to be a bit careful here about making sure these internal functions have leading underscores otherwise we are adding to the public API

```
In [1]: from scipy.special import stirling2_inexact

In [2]:
```

Reply:
This already came up in the review comments. @rlucas7 is going to change this to have just one private `_stirling2_inexact` before marking this ready for review. At the moment, all of these functions are floating around so they can be easily benchmarked against each other.

Reply:
no worries, thanks for the reminder @j-bowhay